src/components/media/articleBody.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { ReactNode } from 'react';
+import { css, SerializedStyles } from '@emotion/core'
+import { adStyles, darkModeCss } from 'styles';
+import { background, neutral } from '@guardian/src-foundations/palette';
+import { getPillarStyles, Pillar, PillarStyles } from 'pillar';
+
+const ArticleBodyStyles = css`
+    position: relative;
+    clear: both;
+    background: ${background.inverse}
+    
+    color: ${neutral[86]};
+
+    ${adStyles}
+`;
+
+const ArticleBodyDarkStyles = ({ inverted }: PillarStyles): SerializedStyles => darkModeCss`
+    background: ${background.inverse};
+    color: ${neutral[86]};
+    a {
+        color: ${inverted};
+    }
+    p:last-child {
+        margin-bottom: 0;
+        padding-bottom: 1em;
```

Comment:
Source provides this through `remSpace`, imported from `@guardian/src-foundations`:

```suggestion
        padding-bottom: ${remSpace[4]};
```